Lib/test/test_peepholer.py

```diff
@@ -1150,10 +1150,11 @@ def get_insts(lno1, lno2, op1, op2):
                 lno1, lno2 = (4, 5)
                 with self.subTest(lno = (lno1, lno2), ops = (op1, op2)):
                     insts = get_insts(lno1, lno2, op1, op2)
+                    op = 'JUMP' if 'JUMP' in (op1, op2) else 'JUMP_NO_INTERRUPT'
                     expected_insts = [
                         ('LOAD_NAME', 0, 10),
                         ('NOP', 0, 4),
-                        (op2, 0, 5),
+                        (op, 0, 5),
```

Comment:
The expected output was wrong here (and there was a bug) - in the case of 'JUMP' --> 'JUMP_NO_INTERRUPT' we were previously ending up with 'JUMP_NO_INTERRUPT', but we want it to be 'JUMP'.